pyproject.toml

```diff
@@ -47,6 +49,7 @@ known_third_party = [
     "warp",
     "carb",
     "Semantics",
+    "torchvision"
```

Comment:
Hmm why is this under imports from Isaac Sim? It should be in the same list as torch?

Reply:
is implicitly included in isaac lab, I think shadow hand stuff uses torch vision too, was preinstalled in my env

moved up to be same list as torch